python/src/DistributionImplementation_doc.i.in

```diff
@@ -417,6 +415,44 @@ OT_Distribution_computeDensityGenerator_doc
 
 // ---------------------------------------------------------------------
 
+%define OT_Distribution_computeLogDensityGenerator_doc
+"Compute the log of the density generator.
+
+Parameters
+----------
+y : float,
+    Input.
+
+Returns
+-------
+p : float
+    The value of the log-density generator function at :math:`y`.
+
+Notes
+-----
+The density generator is the function :math:`\phi` such that the probability density function
+of the elliptical distribution rewrites:
+
+.. math::
+
+    f_{\vect{X}}(\vect{x}) =
+        \phi\left(\Tr{\left(\vect{x} - \vect{\mu}\right)}
+                      \mat{\Sigma}^{-1}
+                      \left(\vect{x} - \vect{\mu}\right)
+            \right),
+        \quad \vect{x} \in \supp{\vect{X}}
+
+This function only exists for elliptical distributions.
+
+See Also
+--------
+isElliptical, computePDF"
+%enddef
+%feature("docstring") OT::DistributionImplementation::computeLogDensityGenerator
```

Comment:
this must be attached to EllipticalDistribution (and go into EllipticalDistribution_doc.i.in)

Reply:
Yes I did it.